### PR TITLE
[now-cli] Add pagination for `now domains ls`

### PR DESCRIPTION
**Pull request by anatrajkovska**
This PR implements pagination when running the command to list domains for a user/team.

The way it works is first you run the command `now domains ls` to retrieve the most recent 20 domains, after which, if you have more than 20 domains created, at the bottom bellow the table there will be a message which guides the user how to fetch the next page.

The message is in the following format:

```
> To display the next page use the flag --next 1584396947215
```

An example to fetch the next page is the command:

```
now domains ls --next 1584396947215
```

Where `1584396947215` is the timestamp of the last domains returned in the table.

[PRODUCT-2065]

[PRODUCT-2065]: https://zeit.atlassian.net/browse/PRODUCT-2065

### CODE REVIEW

File: packages/now-cli/src/commands/domains/index.ts [hunk 1]
```diff
@@ -83,7 +83,8 @@ export default async function main(ctx: NowContext) {
       '--cdn': Boolean,
       '--code': String,
       '--no-cdn': Boolean,
-      '--yes': Boolean
+      '--yes': Boolean,
+      '--next': Number,
```
Comment (lucleray) [inline packages/now-cli/src/commands/domains/index.ts:94]: Could you add the `--next` flag to the help page as well?

File: packages/now-cli/src/commands/domains/ls.ts [hunk 2]
```diff
@@ -48,16 +56,24 @@ export default async function ls(
...
-    `${plural('domain', domains.length, true)} found under ${chalk.bold(
-      contextName
-    )} ${chalk.gray(lsStamp())}\n`
+    `Domains found under ${chalk.bold(contextName)} ${chalk.gray(lsStamp())}\n`
   );
   if (domains.length > 0) {
     console.log(`${formatDomainsTable(domains)}\n`);
   }
 
+  if (domains.length === 20) {
```
Comment (styfle) [inline packages/now-cli/src/commands/domains/ls.ts]:
```suggestion
  if (pagination  && domains.length === 20) {
```

File: packages/now-cli/src/commands/domains/index.ts [hunk 3]
```diff
@@ -85,6 +92,7 @@ export default async function main(ctx: NowContext) {
       '--no-cdn': Boolean,
       '--yes': Boolean,
       '--next': Number,
+      '-N': '--next',
```
Comment (styfle) [inline packages/now-cli/src/commands/domains/index.ts:95]: Let's remove `-N` like I mentioned in the other PR.

Comment (anatrajkovska) [reply packages/now-cli/src/commands/domains/index.ts:95]: @styfle Please check https://github.com/zeit/now/pull/3915#discussion_r395814474

**Review by styfle — APPROVED** [review]

**Pull request merged**